guide/using-the-e-puck-robot.md

```diff
@@ -1,10 +1,18 @@
 ## Using the e-puck robot
 
-In this section, you will learn how to use Webots with the ([e-puck robot](#the-e-puck-robot-at-work)).
+%figure "The e-puck robot"
+
+![e-puck.png](images/e-puck.png)
+
+%end
+
 E-puck is a miniature mobile robot originally developed at EPFL for teaching purposes by the designers of the successful Khepera robot.
 The hardware and software of e-puck is fully open source, providing low level access to every electronic device and offering unlimited extension possibilities.
+
+The model includes support for the differential wheel motors (encoders are also simulated, as position sensors), the infra-red sensors for proximity and light measurements, the accelerometer, the camera, the 8 surrounding LEDs, the body and front LEDs; the other e-puck devices are not yet simulated in the current model.
```

Comment:
Why not put this paragraph into the `E-puck model` section?

Reply:
You could also add the "ground sensors extension" (this is optional), and the bluetooth communication device (This is modeled using Emitter / Receiver devices)

Reply:
A short description of the model is present in the last 3 standardized documentation pages.
https://www.cyberbotics.com/doc/guide/using-the-pioneer-3-at-robot